14.09.2020 12:37:09 Checking for system tray...
14.09.2020 12:37:09 Starting pgAdmin4 server...
14.09.2020 12:37:09 Creating server object, port:50381, key:bd734ac3-b2d0-4398-abe1-4c4e8a3af22b, logfile:C:/Users/70aan/.pgAdmin4.1057243102.log
14.09.2020 12:37:09 Python Path: C:/Program Files/PostgreSQL/12/pgAdmin 4/venv/Lib/site-packages;C:/Program Files/PostgreSQL/12/pgAdmin 4/venv/DLLs;C:/Program Files/PostgreSQL/12/pgAdmin 4/venv/Lib
14.09.2020 12:37:09 Python Home: C:/Program Files/PostgreSQL/12/pgAdmin 4/venv
14.09.2020 12:37:09 Initializing Python...
14.09.2020 12:37:09 Python initialized.
14.09.2020 12:37:09 Adding new additional path elements
14.09.2020 12:37:09 Redirecting stderr...
14.09.2020 12:37:09 stderr redirected successfully.
14.09.2020 12:37:09 Initializing server...
14.09.2020 12:37:09 Webapp Path: C:/Program Files/PostgreSQL/12/pgAdmin 4/web/pgAdmin4.py
14.09.2020 12:37:09 Server initialized.
14.09.2020 12:37:09 Starting Server Thread...
14.09.2020 12:37:09 Open the application code and run it.
14.09.2020 12:37:09 Set the port number, key and force SERVER_MODE off
14.09.2020 12:37:09 PyRun_SimpleFile launching application server...
14.09.2020 12:37:10 Application Server URL: http://127.0.0.1:50381/?key=bd734ac3-b2d0-4398-abe1-4c4e8a3af22b
14.09.2020 12:37:10 The server should be up, we'll attempt to connect and get a response. Ping the server
14.09.2020 12:37:32 Attempt to connect one more time in case of a long network timeout while looping
14.09.2020 12:37:33 Everything works fine, successfully started pgAdmin4.
